static/js/composebox_typeahead.js

```diff
@@ -137,6 +141,8 @@ function query_matches_emoji(query, emoji) {
     // replaces spaces with underscores
     query = query.toLowerCase();
     query = query.split(" ").join("_");
+    query = clean_query(query);
```

Comment:
This could also be using `clean_query_lowercase`; I'm merging the next with that change added.